crates/astria-celestia-jsonrpc-client/k8s/static/NJ3XILLTMVRXEZLUFZVHO5A

```diff
@@ -0,0 +1 @@
+{"body":"/ldpn/YNAogf0Hh/62qoLH+bGBIK15+qnnLZYwd/DVk="}
```

Comment:
what is this?

Reply:
See the readme under the testing section. This is to get hard coded tokens.

Reply:
Removed that bit. Found a better solution.